node/src/database/config_dumper.rs

```diff
@@ -217,15 +228,25 @@ mod tests {
             "config_dumper",
             "dump_config_does_not_migrate_obsolete_database",
         );
-        let conn = bring_db_0_back_to_life_and_return_connection(&data_dir.join(DATABASE_FILE));
+        let chain_specific_dir = add_chain_specific_directories(Chain::PolyMainnet, &data_dir);
+        let created_dir = create_dir_all(&chain_specific_dir); //
+        if created_dir.unwrap() != () {
+            let x: Result<i32, &str> = Err("Could not create chain directory inside dump_config_does_not_migrate_obsolete_database home/MASQ directory");
+            assert_eq!(x.is_ok(), false);
+        }
+        let conn =
+            bring_db_0_back_to_life_and_return_connection(&chain_specific_dir.join(DATABASE_FILE));
         let dao = ConfigDaoReal::new(Box::new(ConnectionWrapperReal::new(conn)));
         let schema_version_before = dao.get("schema_version").unwrap().value_opt.unwrap();
         assert_eq!(schema_version_before, "0");
         let mut holder = FakeStreamHolder::new();
         let args_vec: Vec<String> = ArgsBuilder::new()
             .param("--data-directory", data_dir.to_str().unwrap())
             .param("--real-user", "123::")
-            .param("--chain", TEST_DEFAULT_CHAIN.rec().literal_identifier)
+            .param(
+                "--chain",
+                masq_lib::constants::POLYGON_MAINNET_FULL_IDENTIFIER,
```

Comment:
`Chain::PolyMainnet.rec().literal_identifier` is more idiomatic for this project.